src/librustdoc/passes/stripper.rs

```diff
@@ -70,8 +71,8 @@ impl<'a> DocFolder for Stripper<'a> {
 
             clean::ImplItem(..) => {}
 
-            // tymethods/macros have no control over privacy
-            clean::MacroItem(..) | clean::TyMethodItem(..) => {}
+            // tymethods have no control over privacy
+            clean::TyMethodItem(..) => {}
 
             // Proc-macros are always public
```

Comment:
Should this also be changed for proc macros? Can you add a test? You can add a dependency using // aux-build in a UI test.

Reply:
Probably not? Proc macros are always public, as far as I can tell. An import of a proc macro could still be private, but I'm not sure if that needs to be handled separately.